net-im/qtox/qtox-1.3.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils qmake-utils
+
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/tux3/qtox"
+SRC_URI="https://github.com/tux3/qTox/archive/v${PV}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk X"
+
+# needed, since tarball provided by github extracts to `qTox`
+S="${WORKDIR}/qTox-${PV}"
+
+DEPEND="
+	dev-db/sqlcipher
+	dev-qt/linguist-tools:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5[gif,jpeg,png,xcb]
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtsql:5
+	dev-qt/qtsvg:5
+	dev-qt/qtxml:5
+	media-gfx/qrencode
+	media-libs/openal
+	>=media-video/ffmpeg-2.6.3[webp,v4l]
+	gtk? (	dev-libs/atk
+			dev-libs/glib:2
+			x11-libs/gdk-pixbuf[X]
+			x11-libs/gtk+:2
+			x11-libs/cairo[X]
+			x11-libs/pango[X] )
+	net-libs/tox[av]
+	X? ( x11-libs/libX11
+		x11-libs/libXScrnSaver )
+"
+RDEPEND="${DEPEND}"
+
+pkg_pretend() {
+	if [[ ${MERGE_TYPE} != binary ]]; then
+		if [[ $(tc-getCXX) == *g++ ]] ; then
+			if [[ $(gcc-major-version) == 4 && $(gcc-minor-version) -lt 8 || $(gcc-major-version) -lt 4 ]] ; then
+				eerror "You need at least sys-devel/gcc-4.8.3"
+				die "You need at least sys-devel/gcc-4.8.3"
+			fi
+		fi
+	fi
+}
+
+src_prepare() {
+	epatch_user
+}
+
+src_configure() {
+	use gtk || NO_GTK_SUPPORT="ENABLE_SYSTRAY_STATUSNOTIFIER_BACKEND=NO ENABLE_SYSTRAY_GTK_BACKEND=NO"
```

Comment:
This is a matter of opinion, but it might be cleaner to use ``usex`` instead:

```
eqmake5 \
		...
		ENABLE_SYSTRAY_STATUSNOTIFIER_BACKEND=$(usex gtk) \
		...
```

If qmake CONTAINS is case-sensitive, that would be ``$(usex gtk YES NO)`` then.

If it is preferred to keep it as-is, the variables should probably be local.